app/services/jira_client/repository.rb

```diff
@@ -36,5 +36,10 @@ def fetch_response
         request.params = @request_params
       end
     end
+
+    def raised_exception(exception)
+      ExceptionHunter.track(StandardError.new("Jira key #{@jira_board.jira_project_key} not found"),
```

Comment:
also, the ExceptionHunter `track` method accepts `custom_data` if the only thing that we want is to add the `jira_project_key`

https://github.com/rootstrap/exception_hunter/blob/master/lib/exception_hunter/tracking.rb#L21

Reply:
IDK.. seems like the correct implementation would be just

`ExceptionHunter.track(exception, custom_data: { jira_project_key: @jira_board.jira_project_key })`

so that we maintain the actual exception instead of creating a new one

Reply:
With @santiagovidal were taking about having a clear custom message to avoid looking at logs. Thoughts?

BTW.. both approaches can be done with custom exception as you pointed out 🙌

Reply:
Makes sense, let's go with the custom exception then